.github/workflows/mobile-beta-builds.yml

```diff
@@ -0,0 +1,174 @@
+        uses: actions/download-artifact@v4
+        with:
+          name: app-release.aab
+          path: ${{ github.workspace }}
+
+      - name: Download IPA artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: app-release.ipa
+          path: ${{ github.workspace }}
+
+      - name: Upload IPA to Firebase App Distribution
+        id: firebase_upload_ipa
+        uses: wzieba/Firebase-Distribution-Github-Action@v1
+        with:
+          appId: ${{secrets.FIREBASE_APP_ID_IOS}}
+          serviceCredentialsFileContent: ${{ secrets.CREDENTIAL_FILE_CONTENT }}
+          groups: leatherhood
+          file: ${{ github.workspace }}/app-release.ipa
+          releaseNotes: ${{ needs.update-changelog.outputs.changelog }}
+
+      - name: Upload AAB to Firebase
+        id: firebase_upload_aab
+        uses: wzieba/Firebase-Distribution-Github-Action@v1
+        with:
+          appId: ${{secrets.FIREBASE_APP_ID}}
+          serviceCredentialsFileContent: ${{ secrets.CREDENTIAL_FILE_CONTENT }}
+          groups: leatherhood
+          file: ${{ github.workspace }}/app-release.aab
+          releaseNotes: ${{ needs.update-changelog.outputs.changelog }}
+
+  create-beta-release:
+    needs:
+      - upload-to-firebase
+      - update-changelog
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Create GitHub Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.LEATHER_BOT }}
+        with:
+          tag_name: ${{ github.ref_name }}
+          release_name: '@leather/mobile: ${{ github.ref_name }}'
+          body: |
+            ## Version: ${{ github.ref_name }}
+            [Android - Firebase Console Link](${{ needs.upload-to-firebase.outputs.firebase_console_uri_android }})
+            [iOS - Firebase Console Link](${{ needs.upload-to-firebase.outputs.firebase_console_uri_ios }})
+
+            ---
+
+            ## What's Changed?
+            ${{ needs.update-changelog.outputs.changelog }}
+          draft: true
+
+      - name: Download AAB artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: app-release.aab
+          path: ${{ github.workspace }}
+
+      - name: Download IPA artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: app-release.ipa
+          path: ${{ github.workspace }}
+
+      - name: Upload AAB to Release
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.LEATHER_BOT }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ${{ github.workspace }}/app-release.aab
+          asset_name: ${{ github.ref }}.aab
+          asset_content_type: application/vnd.android.package-archive
+
+      - name: Upload IPA to Release
+        uses: actions/upload-release-asset@v1
```

Comment:
this is apparently an unmaintained action and https://github.com/softprops/action-gh-release is recommended

Reply:
Yeah good catch, a bit hesitant to add that change to this PR since that action is published and working as is.